packages/ipfs-multipart/src/it-multipart.js

```diff
@@ -0,0 +1,195 @@
+'use strict'
+
+const bIndexOf = require('buffer-indexof')
+const parseHeaders = require('parse-headers')
+
+module.exports = multipart
+
+async function* multipart(stream, boundary) {
+  if (!boundary) {
+    if (
+      stream &&
+      stream.headers &&
+      stream.headers['content-type'] &&
+      stream.headers['content-type'].includes('boundary')
+    ) {
+      boundary = stream.headers['content-type'].split('boundary=')[1].trim()
+    } else {
+      throw new Error('Not a multipart request')
+    }
+  }
+
+  boundary = `--${boundary}`
+  const headerEnd = Buffer.from('\r\n\r\n')
+
+  // allow pushing data back into stream
+  stream = prefixStream(stream)
+
+  // consume initial boundary
+  await consumeUntilAfter(stream, Buffer.from(boundary))
+
+  for await (const chunk of stream) {
+    console.log('multipart chunk', chunk)
```

Comment:
Copied and pasted the contents of the [`it-multipart` package](https://www.npmjs.com/package/it-multipart) into this repo so that there is a record of the log statements that were added when testing IPFS in React Native

Reply:
There will be a linting error unless we add the deps of that package as deps for `ipfs-multipart` https://github.com/achingbrain/it/blob/42faa5aeb9de0e07c956d73020bd74fd08e3029d/packages/it-multipart/package.json#L17-L20